readme.md

```diff
@@ -39,64 +39,90 @@ $ npm -v
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
 * **Your Answer:**
+just cloning a repository - make no linking back to your github repo, no updaing when things are changed
+forking and cloning - linking to your github repo, updates can be pushed to online repo 
+
 
----
 
 - [ ] Run `npm init -y` from the command line
 
 * **Question:** What does `npm init` do? How does the `-y` flag modify that command?
 
 * **Your Answer:**
 
----
+npm init 
+to initialize a project
+promt for input from user
+project's name,
+project's version,
+project's description,
+
+npm init -y
+-y stands for yes
+generate an empty npm project without going through an interactive process
 
 - [ ] Take a look at the file that was generated by the previous command
 
 * **Question:** What is the purpose of the following keys? "name", "scripts", "license"
 
 * **Your Answer:**
 
----
+name: project's name
+scripts: to automate repetitive tasks, put them in scripts if you dont want to type long line into command promt
+license: project's license set to ISC Internet Systems Consortium license
 
 - [ ] Create a `.gitignore` file
 
 * **Question:** What is the purpose of the `.gitignore` file? What is the significance of a "dot-file?"
 
 * **Your Answer:**
 
----
+.gitignore
+contain files that to be ignore: pets.json
+not get push/update to GitHub
+files that change all the time while testing/building
+
+dot-file
+configuration files 
+hidden files
 
 - [ ] Create an `index.js` file with the following contents: `console.log('Hello, Node!')`
 
 * **Question:** From the command line, how can you run this file?
 
 * **Your Answer:**
 
----
+node index.js
 
 - [ ] Run `npm test` from the command line
 
 * **Question:** What happens and how is this related to what is in the `package.json` file? 
 
 * **Your Answer:**
 
----
+ "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
This answer doesn't describe what is happening. When you run `npm test`, the value of test gets run. So, in this case, we get a message printed to the screen and then our program exits.